server/container/guice/cassandra-ldap-guice/src/main/java/org/apache/james/data/LdapUsersRepositoryModule.java

```diff
@@ -57,13 +56,9 @@ public LdapUsersRepositoryConfigurationPerformer(ConfigurationProvider configura
         }
 
         @Override
-        public void initModule() {
-            try {
-                usersRepository.configure(configurationProvider.getConfiguration("usersrepository"));
-                usersRepository.init();
-            } catch (Exception e) {
-                Throwables.propagate(e);
-            }
+        public void initModule() throws Exception {
```

Comment:
Ok if initModule throw a Exception can we do something better than halting ?
If no I guess we don't need checked exception

Reply:
See my answer to Raphael.